src/panels/lovelace/components/hui-marquee.ts

```diff
@@ -41,42 +34,58 @@ class HuiMarquee extends LitElement {
     }
 
     return html`
-      <div>${this.text}</div>
-      ${this._interval
-        ? html`
-            <div>${this.text}</div>
-          `
-        : ""}
+      <div class="marquee">
```

Comment:
Would it be better to just use `:host`. `display: block` in the css and then change the updated to use `this` instead of querying for the class